R/methods.R

```diff
@@ -75,3 +75,60 @@ quantile <- function(d, p, ...) {
   ellipsis::check_dots_used()
   UseMethod("quantile")
 }
+
+#' Compute the likelihood of a probability distribution given data
+#'
+#' @param d A probability distribution object such as those created by
+#'   a call to [bernoulli()], [beta()], or [binomial()].
+#' @param x A vector of data to estimate the parameters of the
```

Comment:
We're not estimating parameters of  a distribution here, but calculating a likelihood / joint density.

Reply:
Sorry that's from copy/pasting. I'll do a proper clean up and reorg of the docs - they're mostly thrown together.

Reply:
This has been updated to "A vector of data to compute the likelihood."